api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java

```diff
@@ -108,18 +107,22 @@ public ReportDefinition constructReportDefinition() {
     rd.setName(getName());
     rd.setDescription(getDescription());
     rd.setParameters(txPvlsDataset.getParameters());
-    rd.addDataSetDefinition("N", Mapped.mapStraightThrough(txNewDataset.constructTxNewDataset()));
-    rd.addDataSetDefinition(
-        "C", Mapped.mapStraightThrough(txCurrDataset.constructTxCurrDataset(true)));
-    rd.addDataSetDefinition("P", Mapped.mapStraightThrough(txPvlsDataset.constructTxPvlsDatset()));
+    //    rd.addDataSetDefinition("N",
```

Comment:
Please re enable the other datasets into the report definition

Reply:
Done